src/sdk.ts

```diff
@@ -71,6 +71,10 @@ export async function getAllConfig(): Promise<object> {
   return request.getKVAll('config/get-all')
 }
 
+export async function deleteConfig(key: string): Promise<string> {
```

Comment:
Won't this be returning a `Promise<boolean>`?

Reply:
Ahh you are right, I forgot about that :D fixing, thanks!